pkg/controller/pxc/tls.go

```diff
@@ -97,24 +102,24 @@ func (r *ReconcilePerconaXtraDBCluster) createSSLByCertManager(cr *api.PerconaXt
 					Kind:  issuerKind,
 					Group: issuerGroup,
 				},
-				Duration:    &metav1.Duration{Duration: pxctls.DefaultValidity},
+				Duration:    duration,
```

Comment:
so when i set validity duration all certs will be valid for this duration, including CA. i wonder if this will make rotation harder when these certs are expired. in PSMDB operator, we don't use this field for CA cert and hardcode 365 days for its validity duration (i don't think that approach is correct either bc what happens if user specifies 366 days for cert validity?). maybe we need to have a separate field for CA validity duration?

@hors wdyt?